dom-creation/dom-create-quiz-notes.md

```diff
@@ -0,0 +1,48 @@
+# dom-creation-quiz-notes
+
+## Quiz Questions
+
+Answer the following questions in the provided markdown file before turning in this exercise:
+
+- Does the `document.createElement()` method insert a new element into the page?
+  yes, it creates a new element with the specified tag name
+- How do you add an element as a child to another element?
+  appendChild()
+- What do you pass as the arguments to the `element.setAttribute()` method?
+  it will take 2 arguments, first being a string of the attribute you are trying to create or update (class, id, type, etc.) and 2nd is the value you would like to assign to that attribute
+- What steps do you need to take in order to insert a new element into the page?
+  use the document.createElement(), to create a new element with the specified tag name
```

Comment:
Followed by setting the needed attributes, and concluded it by adding the new element as a child of an existing element in the DOM tree.